docs/developers/talawa-api/mutations.md

```diff
@@ -16,14 +18,16 @@ There are mainly 3 types of User types in talawa -
 3. `USER`
 
 ### Mutations Permitted by Roles.
+
 The limits of each role's use of mutations are stated below:
+
 #### `SUPERADMIN` -
 
-These are the Mutations exclusive to the `SUPERADMIN` type. Please note that a few of these mutations also have a few extra criteria with who can access them .For eg. except `createOrganization` mutation all other require that SUPERADMIN should also be the creator of the orgnanization.
+These are the Mutations exclusive to the `SUPERADMIN` type. Please note that a few of these mutations also have a few extra criteria with who can access them. For example, except `createOrganization` mutation all other require that SUPERADMIN should also be the creator of the orgnanization.
 
 1. `blockPluginCreationBySuperadmin`
 2. `createAdmin`
-3. `createOrganization` - When a superadmin creates and organization , he/she will aso be one of the `admins` and `creator` of that Organization. Therefore, the Mutations in the next Sections also applies to Super Admin.  
+3. `createOrganization` - When a superadmin creates and organization, he/she will also be one of the `admins` and `creator` of that Organization. Therefore, the Mutations in the next Sections also applies to Super Admin.
```

Comment:
remove the s at the end of applies

Reply:
that means I should put "apply" right?

Reply:
@jmalisha That's correct, thanks for catching that.